polaris-tokens/src/palettes.ts

```diff
@@ -0,0 +1,86 @@
+export const palettes = {
```

Comment:
**thought:** Should these be individual exports? e.g. `export const gray = {...}`